isthmus/src/test/java/io/substrait/isthmus/SubqueryPlanTest.java

```diff
@@ -176,8 +176,6 @@ public void notInPredicateCorrelatedSubquery() throws IOException, SqlParseExcep
             .getScalarFunction()
             .getArguments(0)
             .getValue()
-            .getCast()
-            .getInput()
```

Comment:
As the `InPredicate` didn't return a boolean, a cast was being added here. This is gone now.